frontend/src/components/PlantList.tsx

```diff
@@ -0,0 +1,14 @@
+import {Plant} from "./Plant";
+
+
+export default function PlantList(props: { plants: Plant[] }) {
+    return (
+        <>
+            <h2>Pflanzen</h2>
+            <ul>
+                {props.plants.map(plant => <li key={plant.id}> {plant.name}</li>)}
```

Comment:
Das Leerzeichen könnte raus:
```suggestion
                {props.plants.map(plant => <li key={plant.id}>{plant.name}</li>)}
```